libnmstate/nm/dns.py

```diff
@@ -24,6 +24,8 @@
 from libnmstate.dns import DnsState
 from libnmstate.error import NmstateInternalError
 from libnmstate.nm import active_connection as nm_ac
+from libnmstate.nm import ipv4
+from libnmstate.nm import ipv6
```

Comment:
This will cause loop import problem. Even that works in python3, but let's not do that.

Reply:
In that case I will need to move the dns utility function to the NmProfile or to ipv4/ipv6 file. Will take a look.

Reply:
Done!